src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php

```diff
@@ -47,11 +46,6 @@ class UniqueEntityValidatorTest extends AbstractConstraintValidatorTest
      */
     protected $repository;
 
-    protected function getApiVersion()
```

Comment:
shoudln't this be kept for lowest deps (which would use the 2.x validator component) ?